strawberry/chalice/graphiql.py

```diff
@@ -0,0 +1,23 @@
+import functools
+from os.path import abspath, dirname, join
+
+
+@functools.lru_cache()
+def render_graphiql_page() -> str:
+    """
+    Loads the graphiql html file into a string and returns it. Replacing subscription
+    enabled as false, this is because this chalice integration does not currently support
+    subscriptions. This function returns a static result, so cache it in ram, saving us
+    from loading the file from disk each time.
+    Returns:
+        A cached string containing a static graphiql page.
+    """
+    dir_path = abspath(join(dirname(__file__), ".."))
+    graphiql_html_file = f"{dir_path}/static/graphiql.html"
+
+    html_string = None
+
+    with open(graphiql_html_file, "r") as f:
+        html_string = f.read()
```

Comment:
Can we switch this to using modern `pathlib`?

Reply:
Happy to. I was keeping it in the same style as the flask plugin (copy paste job) I'll change it now.

Reply:
Yeah everyone seems to be copying that flask plugin, and every review I have to ask for many of the same changes ahaha. I should stop being lazy and go fix the flask plugin too